cluster-script/benchmark.sh

```diff
@@ -82,6 +82,8 @@ for S in $NETWORK; do
     VARS+=('ab,nginx,$CORES,HTTP-Req/s' 'ab,nginx,$CPUS,HTTP-Req/s')
   elif [ "$S" = fortio ]; then
     VARS+=('fortio,fortio,-c 20 -qps=2000 -t=60s,p999 latency ms' 'fortio,fortio,-grpc -s 10 -c 20 -qps=2000 -t=60s,p999 latency ms')
+  elif [ "$S" = wrk2 ]; then
+    VARS+=('wrk2-benchmark,nginx,-d 60s -c 20 -t 20 -R 2000,p999 latency ms')
```

Comment:
Can we make -c / -t equal to the number of CPU threads on the FIXEDX86NODE?

Reply:
Done